src/main/java/uk/gov/hmcts/darts/ws/CodeAndMessage.java

```diff
@@ -9,6 +9,7 @@ public enum CodeAndMessage {
 
     ERROR("500", null),
     INVALID_XML("400", "Invalid XML Document"),
+    INVALID_JSON("400", "Invalid JSON Document"),
```

Comment:
Not sure about this.  This enum represents the errors that can be returned on the SOAP ws

Reply:
Good spot. My mistake i am no longer using this